src/services/AuthorizationRequestProcessor.ts

```diff
@@ -47,10 +47,11 @@ export class AuthorizationRequestProcessor {
 
 		if (session != null) {
 			if (session.expiryDate < absoluteTimeNow()) {
+				this.logger.info({ message: `Session with session id: ${sessionId} has expired`})
```

Comment:
Shall we append the sessionId to all logs with `logger.appendKeys({ sessionId })` once we've got it?